.kitchen.yml

```diff
@@ -3,7 +3,7 @@
 driver:
   name: docker
   hostname: kitchen-salt.ci.local
-  use_sudo: false
+  use_sudo: true
```

Comment:
This should be false.

If tests are failing on your local box, add your user to the docker group.

Reply:
My bad.